workers/setup.cfg

```diff
@@ -12,13 +11,14 @@ inline-quotes = double
 max-line-length = 88
 
 per-file-ignores =
-    # TODO: Fix issues causing those warnings
-    ohsome_quality_analyst/indicators/mapping_saturation/sigmoid_curve.py:N806,N802,N803,N801
+    # N806: variable in function should be lowercase
```

Comment:
Why is this line at the top?

Reply:
You mean the explanation of the `N806` error? Its there so one knows what kind of issues are ignored per file by flake8.